helsingborg-stad/helsingborg-io-sls-api
# Fix/pdf new application

## Explain the changes you’ve made

A bunch of small html-generation fixes for new applications:

* parse housing types
* handle NaN values
* handle radio button "Ja/Nej" booleans
* add travel expenses
* add buscard
* fix some fields still showing when value is undefined

For full fix the forms have to updated with some tags as well (see ClickUp).

## Explain why these changes are made

Fixes for new application flow.

## How to test

Concrete example:

1. Checkout this branch
2. Generate a PDF with lots of values
3. Verify it seems like expected
